subjects/animals/README.md

```diff
@@ -0,0 +1,65 @@
+## Gatecrashers
+
+### Instructions
+
+Modern JavaScript implements high level tools for creating and managing classes, still most of those tools rely on low level functions still present and in use in some libraries and legacy code.
+
+In order to understand how `class` and `inheritance` works you will have to recreate a similar behavior by using only the low level functions provided by the language.
+
+You will have `Animal` which will be the equivalent of a base class and will have the following fields:
+- `canEat`, `canBreath`, `isAlive`: All booleans, set to true.
+- `WhoAmI`: Function that return a string, here it will be `"I'm an animal"`.
+
+The following objects will inherit from `Animal` and add/override fields as follow:
+- `Dog`: add field `canRun` set to `true`.
+- `Bird`: add field `canFly` and `makeEggs` set to `true`.
+- `Dodo`: override `canFly` and `isAlive` to `false`.
+
+All of them will override `WhoAmI`, returning `"I'm a [animal name]"`.
```

Comment:
I' not sure I'm getting it, can you give me a code example with expected behavior?

Reply:
Something similar to this, this might not be perfect, feel free to refactor it.
```js
const Bird = function(name) {
  this.whoAmI = () => {
    return `This is a ${this.constructor.name} and a ${name}`
  } 
}
const bird = new Bird("dodo")
console.log(bird.whoAmI())
```
Also it would be nice to use the other properties as in the description in Jira to have some scripting fun on the exercise.

Reply:
Thanks a lot for the example, with the new version `this` is used in `NameToUppercase()`